src/common.js

```diff
@@ -172,59 +172,57 @@ exports.getUserHome = getUserHome;
 // Returns {'reference': 'string-value', 'bob': false} when passed two dictionaries of the form:
 //   parseOptions({'-r': 'string-value'}, {'r':'reference', 'b':'bob'});
 function parseOptions(opt, map, errorOptions) {
-  if (!map) error('parseOptions() internal error: no map given');
+  // Validate input
+  if (typeof opt !== 'string' && !(opt instanceof Object)) {
```

Comment:
`Object.create(null) instanceof Object === false`, but `Object.create(null)` indeed creates an object. This is the correct way to check that something is an object: 
```js
typeof a === 'object' && a !== null
```
Maybe make a function for this.